pkg/server/conn.go

```diff
@@ -1699,6 +1707,61 @@ func (cc *clientConn) audit(eventType plugin.GeneralEvent) {
 	}
 }
 
+// parseQueryAttributes support query attributes since mysql 8.0.23
+// see https://dev.mysql.com/doc/refman/8.0/en/query-attributes.html
+// https://archive.fosdem.org/2021/schedule/event/mysql_protocl/attachments/slides/4274/export/events/attachments/mysql_protocl/slides/4274/FOSDEM21_MySQL_Protocols_Query_Attributes.pdf
+func (cc *clientConn) parseQueryAttributes(ctx context.Context, data []byte) (pos int, err error) {
```

Comment:
How well does this handle the case when the data is somehow truncated? (e.g. only one of two bytes)